README.md

```diff
@@ -20,6 +20,10 @@ Kubernetes-based application gateway.
 * [Quickstart](https://gateway.envoyproxy.io/latest/tasks/quickstart/) to use Envoy Gateway in a few simple steps.
 * [Roadmap](https://gateway.envoyproxy.io/contributions/roadmap/)
 
+## Compatibility
```

Comment:
can this live under Documentation section above next to `Roadmap` ?

Reply:
OK